src/hostname/hostnamed.c

```diff
@@ -300,7 +301,20 @@ static const char* fallback_chassis(void) {
                 log_debug("Unhandled ACPI PM profile 0x%02x, ignoring.", t);
         }
 
-        return NULL;
+try_devicetree:
+        r = read_one_line_file("/sys/firmware/devicetree/base/chassis-type", &type);
```

Comment:
Maybe add a comment here: 

```c
/* Note that the Devicetree specification uses the very same vocabulary of chassis types as we do, hence we do not need to translate these types:
 * 
 * https://github.com/devicetree-org/devicetree-specification/blob/master/source/chapter3-devicenodes.rst */
```

Because otherwise the next one looking at this will be surprised that we have no translation here...

Reply:
OK, done